cli/src/config/templates.toml

```diff
@@ -165,3 +165,66 @@ if(hidden,
   )
 )
 '''
+
+'label_log_node(content)' = '''
+label("node",
+  coalesce(
+    if(!self, label("elided", content)),
+    if(immutable, label("immutable", content)),
+    if(conflict, label("conflict", content)),
+    if(current_working_copy, label("working_copy", content)),
+    label("normal", content),
+  )
+)
```

Comment:
nit: I would inline `label()` calls to the main template. It looks a bit obscure if labels and symbols are chosen separately (because these conditions are largely identical.)

If there are labels orthogonal to symbols, maybe we can move them to the top level:
```
label("node " ++ if(working_copy, " working_copy") + ..,
  coalesce(
    .. symbol ..
  )
)
```

Reply:
I added a commit on top implementing this suggestion. I'm not the biggest fan (duplicated code and feels a bit messy), but you be the judge there.

Reply:
Thanks, I agree it's a bit messy, but I think it's good as a starting point. (I don't think the current state is the final "default" node templates because people tend to have their opinions.)

nit: maybe we can use `separate(" ", if(..), if(..))`. I'm not sure which one is better, though:
```
if(!self,
  label("elided", "~"),
  label(separate(" ", ..),
    coalesce(
      ..
    )
  )
)
```
or
```
label(
  if(!self, separate(" ", ..)),
  coalesce(
    if(!self, ..),
    ..
  )
)
```

(Note: I'll send a patch to move the top-level `label("node", ..)` to caller just like `label("op_log", ..)`.)

Reply:
I rewrote to shuffle things around a bit and use `separate`. Dealing with the elided case separately made things a tad neater.

Moved a question about the last note to a comment.